### PR TITLE
NUMA distances

### PR DESCRIPTION
**Pull request by akermu**
The ControlStrategy Scatter tries now to pin the tasks as close as
possible based on the distance of the NUMA nodes.

The ControlStrategy Scatter tries now to spread new task across all NUMA
nodes.

### CODE REVIEW

**Comment by jbreitbart** [comment]
Travis CI: Loooks like you are missing a #include <QStringList> somewhere

File: include/AutopinPlus/ControlStrategy.h [hunk 1]
```diff
@@ -162,7 +162,15 @@ class ControlStrategy : public QObject {
 	 * \return The new pinning. The tasks will be pinned accordingly
 	 * by changePinning.
 	 */
-	virtual Pinning getPinning(const Pinning &current_pinning) const;
```
Comment (jbreitbart) [inline include/AutopinPlus/ControlStrategy.h]: Why did you remove the const here?

**Comment by jbreitbart** [comment]
Regarding the CPUInfo class: There is no need to use a class if anything is static anyway. You can just put the function in their namespace and make the private variables static on file scope in the .cpp file.

File: include/AutopinPlus/ControlStrategy.h (hunk 1 above)
Comment (akermu) [inline include/AutopinPlus/ControlStrategy.h]: I need to modify the class variable pinCount in the Scatter implementation of getPinning.
